lib/mongo/server/description.rb

```diff
@@ -445,6 +445,60 @@ def unknown!
       def wire_versions
         min_wire_version..max_wire_version
       end
+
+      # Is this description from the given server.
+      #
+      # @example Check if the description is from a given server.
+      #   description.self?(server)
+      #
+      # @return [ true, false ] If the description is from the server.
+      #
+      # @since 2.0.6
+      def self?(server)
```

Comment:
I might change this name to something else, given that `self` is reserved in Ruby and this might be a bit confusing.